package.json

```diff
@@ -4,9 +4,9 @@
   "description": "Remote new tab (third version)",
   "main": "src/main.js",
   "scripts": {
-    "setup": "npm run clean && npm run build:static",
+    "setup": "npm run clean && npm run build:static && npm run build:offline",
```

Comment:
This `build:offline` needs to run after `build:js` and `build:css` because the files will not have been generated yet, no?